lib/Invoice.js

```diff
@@ -33,7 +33,9 @@ export class Invoice {
     this._options.paid = options.paid
     this._options.comment = options.comment
     this._options.logoImage = options.logoImage
-    this._options.prepaymentInvoice = options.prepaymentInvoice || false
+    this._options.prepaymentInvoice = options.prepaymentInvoice || false,
+    this._options.finalInvoice = options.finalInvoice || false
+    this._options.prepaymentInvoiceNumber = options.prepaymentInvoiceNumber || ''
```

Comment:
I think better to not set default values for these new fields. I checked the `wrapWithElement` and if the value of the field is `null` or `undefined` it will not be serialised so we can save a small bandwidth.